server/configureChangeFeeds/userEmailUpdated.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console, camelcase */
+import processChangeFeedWithAutoReconnect from 'rethinkdb-changefeed-reconnect'
+
+import {connect} from 'src/db'
+import handleConnectionError from './util/handleConnectionError'
+
+const r = connect()
+
+export default function userEmailUpdated(queueService) {
+  processChangeFeedWithAutoReconnect(
+    _getFeed,
+    _getFeedProcessor(queueService),
+    handleConnectionError,
+    {changefeedName: 'user email updated'}
+  )
+}
+
+function _getFeed() {
+  return r.table('users')
+    .changes()
+    .filter(
+      r.row('old_val').ne(null)('email')
```

Comment:
This throws an error. 

```
r.row('old_val').ne(null)
```
...returns a boolean value instead an object where an `('email')` field can be accessed.